apps/api/src/app/db/content/prose/fandom-tag.document.ts

```diff
@@ -0,0 +1,6 @@
+import { FandomTag } from '@dragonfish/shared/models/content';
+import { Document } from 'mongoose';
+
+export interface FandomTagDocument extends FandomTag, Document {
+    readonly _id: string;
+}
```

Comment:
Just thinking aloud here: the name of this alone makes me wonder how much we want to specialize tags. Should have have a base tag interface that both `FandomTag` and `UserTag` inherit from? Should `UserTag` and `FandomTag` even be different? Worth thinking about.